EKrut-Common/src/ekrut/entity/SaleDiscountType.java

```diff
@@ -1,7 +1,6 @@
 package ekrut.entity;
 
 public enum SaleDiscountType {
-	SECOND_IN_HALF,
+	THIRTY_PERCENT_OFF,
 	ONE_PLUS_ONE,
-	TWO_PLUS_ONE
```

Comment:
I have no problem with the changes, due to they don't specify the sales discount I actually thought that implement sales discount by percentage is harder.
By the way, I went throw all changes and it seems that you right, I'll make changes accordingly.

Reply:
The whole 30% thing was just temporary. I was thinking maybe we should add an optional column that specifies the discount percentage? Although I'm not sure how these discounts would be presented in the GUI.